docs/source/migration.mdx

```diff
@@ -116,32 +122,30 @@ The `startStandaloneServer` function accepts two arguments; the first is the ins
 
 | Name  | Description  |
```

Comment:
For tables with multi-paragraph description cells like this, you might find it easier to use an HTML table a la the field tables in the ApolloServer API reference 🤷

Reply:
Oooh great point, I'll do that!